src/renderer/kube-object.store.ts

```diff
@@ -175,14 +175,15 @@ export abstract class KubeObjectStore<T extends KubeObject = any> extends ItemSt
 
       switch (type) {
         case "ADDED":
-        case "MODIFIED":
+        case "MODIFIED": {
```

Comment:
just curious, why `{...}` is needed here?

Reply:
Because of historial reasons, variables in seperate `case` statements are in the same scope as all other variables in the same `switch` statement. 

Namely, the following will fail to compile due to name collisions

```ts
switch (foo) {
    case 0:
        const x = 1;
        break;
    case 1:
        const x = 2;
        break;
}
```

The `{...}` just creates a new block scope to prevent this.